images/krte/variants.yaml

```diff
@@ -3,30 +3,30 @@
 variants:
   experimental:
     CONFIG: experimental
-    GO_VERSION: 1.12.10
+    GO_VERSION: 1.12.12
     K8S_RELEASE: stable
     BAZEL_VERSION: 0.28.1
     UPGRADE_DOCKER: 'true'
   master:
     CONFIG: master
-    GO_VERSION: 1.12.10
+    GO_VERSION: 1.12.12
     K8S_RELEASE: stable
     BAZEL_VERSION: 0.23.2
   '1.16':
     CONFIG: '1.16'
-    GO_VERSION: 1.12.10
+    GO_VERSION: 1.12.12
     K8S_RELEASE: stable-1.16
     BAZEL_VERSION: 0.23.2
   '1.15':
     CONFIG: '1.15'
-    GO_VERSION: 1.12.10
+    GO_VERSION: 1.12.12
     K8S_RELEASE: stable-1.15
     BAZEL_VERSION: 0.23.2
   '1.14':
     CONFIG: '1.14'
-    GO_VERSION: 1.12.10
+    GO_VERSION: 1.12.12
     K8S_RELEASE: stable-1.14
-    BAZEL_VERSION: 0.21.0
+    BAZEL_VERSION: 0.23.2
```

Comment:
what's the rational to bump bazel for an older release here? I thought that should not change?

Reply:
This was needed to support a rules_go bump that was also needed to bump Go. All the go 1.12 versions use the same bazel version, and the rules_go they use is all compatible.